tests/cupyx_tests/test_cutensor.py

```diff
@@ -337,3 +337,133 @@ def test_contraction(self):
                              self.c, mode_c)
         cupy.testing.assert_allclose(self.c, self.c_ref,
                                      rtol=self.tol, atol=self.tol)
+
+
+@testing.parameterize(*testing.product({
+    'dtype_char': ['e', 'f', 'd', 'F', 'D'],
+    'shape': [(30, 40, 30, 35)],
+    'alpha': [0.5, 1.0],
+    'beta': [0.0, 1.0],
```

Comment:
Let's also test different orders 
```suggestion
    'beta': [0.0, 1.0],
    'order': ('C', 'F'),
```
(need to pass `self.order` to `shaped_random`).